content/en/docs/architecture/step-registry.md

```diff
@@ -959,3 +959,59 @@ tests:
   steps:
     workflow: openshift-e2e-disruptive
 {{< / highlight >}}
+
+## VPN connection
+
+For platforms that need access to restricted environments, `ci-operator`
+supports adding a dedicated VPN connection to each test step.  Since this is a
+requirement of specific platforms, it is triggered by special files in the
+[cluster profile(s)]({{< ref "docs/how-tos/adding-a-cluster-profile#what-is-a-cluster-profile" >}})
```

Comment:
Do we have a cluster profile that supports VPN already?

Reply:
Not yet, I use my own to test locally.  But part of the reason is the interface wasn't defined, so this PR is also intended to address that.